views/choral.pug

```diff
@@ -35,10 +35,10 @@ block content
             div.child-card-body
               - var device_data = JSON.parse(child.device_data);
               - for( var entry in device_data )
-                - var data_value = device_data[entry]
+                - var data_value = device_data[entry].toFixed(5)
```

Comment:
Need this because this is the first value. If we don't put this, it could overflow for the 10 seconds it's shown, then gets updated.